sale_option/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+  :alt: License: AGPL-3
+
+Sale Option
+==============
+
+This module provide an options system for sale order line.
```

Comment:
@Viggor  Can you update the description in order to explain to @pedrobaeza  and everyone the purpose of the module